intel_extension_for_transformers/llm/runtime/graph/models/baichuan/baichuan.cpp

```diff
@@ -114,7 +116,9 @@ static bool baichuan_model_eval_internal(model_context& lctx, const model_token*
   }
 
   struct ne_tensor* embd = d_ne_new_tensor_1d(ctx0, NE_TYPE_I32, N);
-  memcpy(embd->data, tokens, N * ne_element_size(embd));
+  for (int i = 0; i < batch_size; ++i) {
```

Comment:
could we use pragma omp  here ? 
to simply parallel do cpy

Reply:
fixed in [f8c62f5](https://github.com/intel/intel-extension-for-transformers/pull/583/commits/f8c62f5623d571640da1bfd20d9927a4d0eaa9e6)